js/levels.js

```diff
@@ -776,7 +776,7 @@ var levels = [
       'tr': '<p>Aynı genişlikte bir grup kolon belirtmek sıkıntıya neden olabilir.</p><p>Neyse ki <code>repeat</code> fonksiyonu bu tarz durumlarda yardım için var.</p><p>Örneğin; daha önce <code>grid-template-columns: 20% 20% 20% 20% 20%;</code> kuralı ile beş adet %20\'lik kolon tanımlamıştık. Bu şu şekilde kısaltılabilir: <code>grid-template-columns: repeat(5, 20%);</code></p><p><code>grid-template-columns</code> ile <code>repeat</code> işlevini kullanarak, her biri % 12.5 genişliğe sahip sekiz kolon oluşturun. Bu şekilde bahçeniz su altında kalmayacaktır.</p> ',
       'fr': '<p>Le fait de spécifier un ensemble de colonnes avec des largeurs identiques peut devenir fastidieux. Heureusement, il y a une fonction <code>repeat</code> pour nous aider.</p><p>Par exemple, nous avons défini précédemment cinq colonnes de 20% avec la règle <code>grid-template-columns: 20% 20% 20% 20% 20%;</code>. Ceci peut être simplifié ainsi <code>grid-template-columns: repeat(5, 20%);</code></p><p>A l\'aide de <code>grid-template-columns</code> avec la fonction <code>repeat</code>, créez huit colonnes avec chacune une largeur de 12.5%. De cette façon, vous ne sur-arrosez pas votre jardin.</p>',
       'de': '<p>Mehrere Spalten mit identischen Breiten anzugeben kann mühsam werden. Zum Glück gibt es eine <code>repeat</code> Funktion, die uns hilft.</p><p>Zum Beispiel haben wir vorher fünf Spalten mit 20% definiert mit der Regel <code>grid-template-columns: 20% 20% 20% 20% 20%;</code>. Dies kann vereinfacht werden zu <code>grid-template-columns: repeat(5, 20%);</code></p><p>Benutze <code>grid-template-columns</code> mit der <code>repeat</code> Funktion um acht Spalten von je 12.5% Breite zu erstellen. So wirst du deinen Garten nicht überwässern.</p>',
-      'nl': '<p>Het opgeven van een stel kolommen met identieke breedtes kan vervelend worden. Gelukkig is er een <code>repeat</code> functie om daarmee te helpen.</p><p>Bijvoorbeeld, we hebben eerder vijf 20% kolommen gedefinieerd met de regel <code>grid-template-columns: 20% 20% 20% 20% 20%;</code>. Dat kan vereenvoudigd worden naar <code>grid-template-columns: repeat(5, 20%);</code></p><p>Gebruik <code>grid-template-columns</code> met de <code>repeat</code> functie, maak acht kolommen met ieder een breedte van 12.5%. Op deze manier zal je tuin niet teveel water krijgen.</p>',
+      'nl': '<p>Het opgeven van een stel kolommen met identieke breedtes kan vervelend worden. Gelukkig is er een <code>repeat</code>-functie om daarmee te helpen.</p><p>We hebben bijv. eerder vijf 20 %-kolommen gedefinieerd met de regel <code>grid-template-columns: 20% 20% 20% 20% 20%;</code>. Dat kan vereenvoudigd worden naar <code>grid-template-columns: repeat(5, 20%);</code></p><p>Gebruik <code>grid-template-columns</code> met de <code>repeat</code>-functie, maak acht kolommen met ieder een breedte van 12.5 %. Op deze manier zal je tuin niet te veel water krijgen.</p>',
```

Comment:
Here also an abbreviation and percentages with a space between the number and the unit.